Cdn case study.md (#191)
NoroffFEU/development-platforms: src/content/docs/CDN/CDN-case-study.md

## Application example

This is a step by step guide on how to use bootstrap CSS with CDN on a new HTML document

1. First start by creatig a new HTML document, Bootstrap requires HTML5 doctype.

> 2. in the `<head>` tag of the HTML file add a `<link>` tag, this link tag is what is going to import Bootsrap using a CDN. for this example JSdelivr is used copy and paste this link and place it in the `<head>` of the HTML document:

Reviewer: Fix the typo
```suggestion
2. In the `<head>` tag of the HTML file, add a `<link>` tag. This link tag is what is going to import Bootstrap using a CDN. For this example, JSdelivr is used to copy and paste this link and place it in the `<head>` of the HTML document:
```

```html
<link
  rel="stylesheet"
  href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
  integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
  crossorigin="anonymous"
/>
```

3. The HTML page should look something like this:

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Bootstrap CDN example</title>
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
      crossorigin="anonymous"
    />
  </head>
  <body></body>
</html>
```

> 4. Bootstrap CSS should now be implemented to test if Bootstrap CSS is implemented correctly copy and paste this code in the `<body>` tag which is shown below:

Reviewer: Fix the tone here
```suggestion
4. To test if Bootstrap CSS has been implemented correctly, copy and paste the following code into the <body> tag:
```

```html
<body><h1 class="bg-primary">My First Heading</h1></body
```

> now open a live server and look at the Heading, if the background is blue Bootstrap was successfully implemented if the background of the Heading is not blue Bootstrap CSS was not implemented correctly.

Reviewer: Fix the tone here
```suggestion
5. Now, open with live server and observe the Heading. If the background is blue, Bootstrap has been successfully implemented. If the background of the Heading is not blue, Bootstrap CSS has not been implemented correctly.
```

Author: Thank you very much for the sugestions, I have changed the text to the suggested grammar and tone

A content delivery network is a collection of servers around the world that are connected to a main server often called the origin server. The servers are strategically placed geographically across different countries and continents, and these servers are called POP (point of presence) servers or edge servers. The main function of these servers is to cache information such as HTML, JavaScript, CSS, and video content closer to the end user so loading time, performance, and reliability are improved, whereas a non-CDN network may take 5 seconds to load heavy website content to an end-user on the other side of the world a CDN network may have a pop server only a few miles from the end user that has cached heavy images and stylesheets so the loading time may be reduced to only 1 second.
![Features](~/assets/CDN/OJ8XXP0.jpg)

## Problem statement

As talked about in the brief history section, a major problem with the state of networking in the 90s was slow speed, poor performance, and sometimes server crashes when there was too much traffic. This was a problem because of multiple factors such as

- The Internet became more popular and grew rapidly and networking couldn't keep up.

> - People connected to websites that were hosted from a great geographical distance which caused long waiting times for the end-user.

Reviewer: @mathiasg12 This is a good write-up. However, while I was reading, I felt like CDN is meant to solve the problem of the past. Can you establish the fact that CDN is a tool in this modern day? Can you demonstrate what the situation is like without/with it?

Author: thanks for the feedback I will add a section about a world with/without CDN

Reviewer: You can use bootstrap to demonstrate

Author: I added some sentences at the end of the section "Main feature and objectives" explaining  a world without and with CDNs I also used bootstrap as an example of modern uses of CDNs
